bulbs/campaigns/templatetags/campaign_tags.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from django import template
+from django.utils.dateparse import parse_datetime
+
+import requests
+from pytz import timezone
+
+
+register = template.Library()
+
+
+@register.inclusion_tag('campaigns/_campaign_pixels.html')
+def campaign_pixels(*args, **kwargs):
+    campaign_id = kwargs['campaign']
+    property_id = kwargs['property']
+
+    params = {'campaign': campaign_id, 'property': property_id}
+    response = requests.get('http://www.tunic.theonion.com/api/v1/pixels', params=params)
+
+    valid_pixels = []
+
+    for pixel in response.json()['results']:
+        now = datetime.now(timezone('UTC'))
+
+        begun = parse_datetime(pixel['campaign_start_date']) < now
```

Comment:
Use <= for begun, generally we've been following convention:

start <= now < end